tests/contrib/botocore/test.py

```diff
@@ -165,6 +165,30 @@ def test_sqs_client(self):
         self.assertEqual(span.service, 'test-botocore-tracing.sqs')
         self.assertEqual(span.resource, 'sqs.listqueues')
 
+    @mock_sqs
+    def test_sqs_send_message_trace_injection(self):
+        sqs = self.session.create_client('sqs', region_name='us-east-1')
+        queue = sqs.create_queue(QueueName='test', Attributes={'DelaySeconds': '5'})
+        Pin(service=self.TEST_SERVICE, tracer=self.tracer).onto(sqs)
+
+        response = sqs.send_message(QueueUrl=queue['QueueUrl'], MessageBody='world')
```

Comment:
Are there any properties in `response` we can assert on to make sure it's working as expected?

Reply:
There's a `MD5OfMessageAttributes` but the process of calculating this checksum is a bit complex for my python knowledge. https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-message-metadata.html#sqs-attributes-md5-message-digest-calculation

I can implement it if the `span` getTag is not enough. Let me know!

Reply:
Ah yeah no I don't think we need to get into the MD5 stuff. It was more just to make sure that we're not breaking the message send. It doesn't look like there's much we can rely on other than not getting an erroneous response from the mock sqs.

However when I was looking at https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html#SQS.Client.send_message I saw `send_message_batch`, does this also get instrumented with this patch? If so we should also have a test with it.

Reply:
It doesn't - I can add that patch as well!